src/components/flowchart/index.js

```diff
@@ -243,7 +243,10 @@ export class FlowChart extends Component {
         });
       });
 
-    this.el.svg.call(this.zoomBehaviour);
+    this.el.svg
+      .call(this.zoomBehaviour)
+      // Prevent default double click zoom.
+      .on('dblclick.zoom', null);
```

Comment:
Yeah the issue here is that it interferes with the metadata panel movement / zoom on click. I'd like to keep this too ideally, but was waiting for design input on the sidebar button which would avoid the need for any fix here.

But we can either wait for that (probably a later PR) or I can look at implementing a workaround now (potentially reimplementing the double click zoom feature and accounting for the metadata panel). I'm not sure how long that would take, but it might be worth it if we are considering releasing this unflagged (else I could put a flag on this line for now).

Reply:
Okay fair enough. If it's necessary to keep this as is for now, I'd recommend adding to the comment, to explain why it's disabled, so that the next dev doesn't come along and wonder why and remove this line.

Also maybe create a new ticket for implementing the workaround or design change etc if necessary, just so we don't lose this through the cracks?

Reply:
Added a better comment here, will add a ticket too.